crates/teloxide/src/dispatching/dispatcher.rs

```diff
@@ -495,6 +495,32 @@ impl<R, Err, Key> Dispatcher<R, Err, Key> {
     }
 }
 
+/// Extension methods for adding `tracing` to [`UpdateHandler`].
+///
+/// This trait is only available if the `tracing` feature is enabled.
+#[cfg(feature = "tracing")]
+pub trait UpdateHandlerTracingExt<E> {
```

Comment:
This should be in its own module, not `dispatching::dispatcher`, but `dispatching::tracing`

Reply:
Moved to new module.